### PR TITLE
Application deploy

### PR DESCRIPTION
**Pull request by Thoth1111**
### Application Deploy:

In this branch, I deployed the portfolio using GitHub pages and modified the Readme file to include the link to the project under the live demo part.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -81,7 +81,7 @@
 
 <!-- LIVE DEMO -->
 
-## 🚀 Live Demo <a name="live-demo"></a>
+## 🚀 Live Demo <a name="https://thoth1111.github.io/Alwyn_Portfolio/"></a>
 
 > Add a link to your deployed project.
```
Comment (Owusu-Desmond) [inline README.md]: - [x] Nice to have your live demo link working :+1:  Please make sure that you put the link to the live demo in the right place. You should not put the link where it is now. I suggest it should be like this :point_down: 
```
## 🚀 Live Demo <a name="live-demo"></a>

- [Live Demo Link](https://thoth1111.github.io/Alwyn_Portfolio/)
```

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @Thoth1111 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the setup of the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Thoth1111) [reply README.md]: Thank you. I have made the change now.

**Review by levy002 — APPROVED** [review]
## Hello @Thoth1111 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Status: APPROVED :fire:  :fire: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**